echo/downloader.py

```diff
@@ -132,19 +133,22 @@ def downloadHQ(self, medias, time, type, courseName, termName):
         self.downloadResource(medias[bestIndex]["s3Url"], time, type, courseName, termName)
 
     def downloadResource(self, url, time, media_type, courseName, termName):
-        dt = date_parser.parse(time)
-        date_formatted = dt.strftime("%m.%d - %H%M")
-
+        try:
+            dt = date_parser.parse(time)
+            date_formatted = dt.strftime("%m.%d - %H%M")
+        except TypeError: # not all lectures have a recording date/time because of manual upload
+            date_formatted = datetime.now().strftime("UNKNOWN TIME. downloaded at %m.%d - %H.%M.%S")
```

Comment:
not sure if this is a good backup name solution, maybe an order which the lecture appears would be better

Reply:
that's a good idea